src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -269,7 +269,6 @@
     extends     ("(if|while)_let_expr") = scoped_let_expr
     extends     (".*_expr") = restricted_expr
     extends     ("(?!ret_type).*_type") = type
-    extends     (".*_stmt") = stmt
```

Comment:
The more I work with our parser, the less I like that `extends` declarations are detached from the actual productions. They are nice if you want to save typing `extends` for a dozen of productions (like expessions). But here we have only three cases, and it is clearer just to specify them inline.